examples/crd-client/Dockerfile

```diff
@@ -14,21 +14,18 @@
 
 # Gather dependencies and build the executable
 FROM golang:1.20.4 as builder
-WORKDIR /go/src/crd-client
+WORKDIR /
 
 COPY ./main.go .
 RUN go mod init && go mod tidy
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o client .
 
 # Create the final image that will run the webhook server for FleetAutoscaler webhook policy
-FROM alpine:3.13.1
-RUN adduser -D -u 1000 client
+FROM gcr.io/distroless/static-debian11:nonroot
 
-COPY --from=builder /go/src/crd-client \
-                    /home/client
+COPY --from=builder /client /
```

Comment:
I do like just copying the binary over; that seems like an improvement from the previous commands. 👍